pulsar-broker/src/test/java/org/apache/pulsar/compaction/CompactedTopicTest.java

```diff
@@ -37,6 +37,7 @@
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.stream.IntStream;
 
+import io.swagger.util.Json;
```

Comment:
Is this the correct import?

Reply:
Oh, sorry. Forget to rebase the master branch.